library/cwm/src/lib/cwm/table.rb

```diff
@@ -119,9 +148,16 @@ def multiselection?
   private
 
     def format_items(items)
-      items.map do |item|
-        Item(Id(item.first), *item[1..-1])
-      end
+      items.map { |i| format_item(i) }
+    end
+
+    # @see #format_items
+    def format_item(item)
+      return Item(Id(item.first), *item[1..-1]) if item.is_a?(Array)
+
+      children = format_items(item.children)
+      open_tag = item.open ? :open : :close
```

Comment:
that symbol needs to be :close**d** , not :close !

Reply:
Wrong symbol fixed.

On the other hand, the sort-keys were already supported  before my changes because each value can be a `Cell` element.